packages/core/content-releases/server/src/controllers/__tests__/release-action.test.ts

```diff
@@ -123,9 +124,20 @@ describe('Release Action controller', () => {
       await releaseActionController.createMany(ctx);
 
       expect(mockCreateAction).toHaveBeenCalledTimes(2);
-      expect(ctx.body.data).toHaveLength(2);
-      expect(ctx.body.meta.totalEntries).toBe(2);
-      expect(ctx.body.meta.entriesAlreadyInRelease).toBe(0);
+
+      expect(ctx.created).toHaveBeenCalledWith(
+        expect.objectContaining({
+          body: expect.objectContaining({
+            data: expect.any(Array), // Ensure data is an array
+            meta: expect.objectContaining({
+              totalEntries: 2,
+              entriesAlreadyInRelease: 0,
+            }),
+          }),
+        })
+      );
+      const firstCallArgument = ctx.created.mock.calls[0][0];
+      expect(firstCallArgument.body.data.length).toBe(2);
```

Comment:
This seems to be expecting this
```
    Expected: ObjectContaining {"body": ObjectContaining {"data": Any<Array>, "meta": ObjectContaining {"entriesAlreadyInRelease": 0, "totalEntries": 2}}}
    Received: {"data": [{"id": 1}, {"id": 1}], "meta": {"entriesAlreadyInRelease": 0, "totalEntries": 2}}

```

Should data not be nested in body in the expectations in this file?

Reply:
yeah, I wrote the test wrong, fixing it now